### ISSUE TITLE
landlord listing - validation of address

### ISSUE THREAD

**Issue by inseokr** [inseokr/MERN_linkedspace#499]
I don't see any validation of address. 

We should introduce a validation similar to the one in 3rd party listing

**Comment by justin-seo** [comment]
this was because it's not being validated from frontend because it's still EJS code. can try validating on backend but because that's patch work I didn't do it. Can add it on the backend side if needed

**Comment by inseokr** [comment]
Yes, it's EJS code. Eventually we will get rid of all EJS codes.
We may need to prioritize tasks on your plate. 
Let's hold this issue for now so that you could focus on other priority issues.
We may get to this when all other MVP items are done.

**Comment by inseokr** [comment]
#500 will be higher priority than this.
